website/docs/components/button/button-code.md

```diff
@@ -5,7 +5,10 @@ tabs: Design('button'), A11y('button-a11y'), API('button-api'), Example('button-
 
 ## Addons
 
-Addons can be installed either by passing the required `tag` to the `addonLeft`/`addonRight` property or by unrending `Button.Addon`/`Button.Text` into the component body. The examples below are the same.
+Addons can be added:
+
+- By passing the required `tag` to the `addonLeft`/`addonRight` property.
+- By unrending `Button.Addon`/`Button.Text` into the component body.
```

Comment:
I'm not sure "unrending" is a word 😅 Let's say "nesting" or something like that :)